packages/gravity-ui-web/src/sass/05-components/03-organisms/_copy-ostentatious.scss

```diff
@@ -0,0 +1,38 @@
+.grav-c-ostentatious-copy {
+  padding-right: $grav-sp-l;
+  padding-left: $grav-sp-l;
+
+  > * {
+    display: block;
+
+    + * {
+      margin-top: $grav-sp-vertical-gap-l;
+    }
+  }
+
+  img {
+    width: 100%;
+    height: auto;
+  }
+
+  @media all and (max-width: grav-breakpoint(medium)) {
+    padding-right: $grav-sp-s;
```

Comment:
We can avoid this `max-width` media query. You can set this padding and the blockquote styles outside of the media query.

Then, below, in the `min-width` media query, where you already alter the blockquote styles as needed, you can update the padding to be large instead of small.